recipes/rio-cogeo/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: rio-cogeo
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/r/rio-cogeo/rio-cogeo-{{ version }}.tar.gz
+  sha256: "05940d0523b3c0d400177b29a2a33712b84881bbeb82e713a1b9f064422591f9"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >3.3
+    - pip
+    - rasterio[s3] >=1.0.9
```

Comment:
Does using square brakets here really work?

Reply:
Nope. That is a remnant from conda-skeleton. I added the proper dependency below. I'm surprised it did not crash the build though.

Reply:
me too!